### PR TITLE
Blog-app Views

### PR DESCRIPTION
**Pull request by ismayito**
### App views
______________________________________________________________________________________________________
#### Summary
______________________________________________________________________________________________________
In this project feature I added the following:
- Added preprocessed HTML file with embedded Ruby code.
- Used layouts and templates for shared content.
- Implement the design from the sneak peek wireframes
- 
Screenshot from 2023-12-14 18-00-20

Screenshot from 2023-12-14 18-00-51
Screenshot from 2023-12-14 18-01-24

Screenshot from 2023-12-14 18-02-02

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,33 @@
-<h1>Shows a specific user</h1>
\ No newline at end of file
+<%= render partial:"user",locals: {user:@user} %>
+<div class="user-bio">
+<p>Bio:</p>
+<p><%= @user.Bio %></p>
+</div>
+<div>
+<% @user.posts.each_with_index do |post,index| %>
```
Comment (gemmen29) [inline app/views/users/show.html.erb]: - [ ] Kindly use the methods created in the `Blog App - processing data in models` to display posts in the `user#show` page as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) and to match the given wireframe.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,40 @@
...
+          <% end %>
+      </p>
+    </div>
+    <% end%>
+    </div>
+<% if post.comments.empty? %> 
+ <div class='none'></div>
+ <% else %>
+  <div class='user-post'>
+  <% post.comments.each do |comment|  %>
```
Comment (gemmen29) [inline app/views/posts/index.html.erb]: - [ ] Kindly use the methods created in the `Blog App - processing data in models` to display comments in the `post#index` page as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) and to match the given wireframe.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ismayito, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,33 @@
...
+          <% end %>
+      </p>
+    </div>
+</div>
+<% end%>
+<% end %>
+</div>
+<section class='button-container'>
+  <button type="button" class="see-all">See all posts</button> 
+</section>
```
Comment (tresorsawasawa) [inline app/views/users/show.html.erb]: - [ **OPTIONAL** ] As required for the the project, kindly ensure that when you click on the "See all Post" button, it opens user's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png).

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @ismayito 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All views render data from db :heavy_check_mark: 
- Used partial file(s) :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**